bokehjs/package.json

```diff
@@ -114,6 +114,7 @@
     "slickgrid": "https://github.com/bokeh/SlickGrid.git#6c1327b"
   },
   "scripts": {
+    "prepare": "node ./prepare.js",
```

Comment:
Is this documented anywhere? I had to find and read this PR in order to build bokeh locally. @mattpap